aspnet/publishing/iis.rst

```diff
@@ -1,116 +1,187 @@
 Publishing to IIS
-=============================
+=================
 
-By `Rick Anderson`_ and `Luke Latham`_
+By `Rick Anderson`_ and `Luke Latham <https://github.com/GuardRex>`_
 
-.. contents:: Sections:
-  :local:
-  :depth: 1
+- `Supported Operating Systems`_
+- `IIS Configuration`_
+- `Install the Microsoft ASP.NET Core Module`_
+- `Install .NET Core on the Server`_
+- `Deploy the Application`_
+- `Configure the Website in IIS`_
+- `Create a Data Protection Registry Hive`_
+- `Common Errors`_
+- `Additional Resources`_
 
-Install the HTTP Platform Handler
------------------------------------
+Supported Operating Systems
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-- Install the HTTP Platform Handler version 1.2 or higher:
+The following operating systems are supported:
 
-  - `64 bit HTTP Platform Handler <http://go.microsoft.com/fwlink/?LinkID=690721>`_
-  - `32 bit HTTP Platform Handler <http://go.microsoft.com/fwlink/?LinkId=690722>`_
+- Windows 7 and newer
+- Windows Server 2008 R2 and newer\*
 
-If you need to enable IIS, see `IIS server configuration`_.
+\*Conceptually, the IIS configuration described in this document also applies to hosting ASP.NET Core applications on Nano Server IIS, but refer to `ASP.NET Core on Nano Server <https://docs.asp.net/en/latest/tutorials/nano-server.html>`_ for specific instructions.
```

Comment:
> That should be `blah http://foo`__ ... correct?

Oops, fixed above.

> You want all inline hyperlinks with the double-underscore? 

Yes. The problem with one underscore is that you are implicitly defining a link target for the friendly text of the link. So `` `foo <http://bar>`_ `` is really the same as:

``` rst
foo_

.. _foo: http://bar
```

For one link this isn't an issue, but if you will get a warning if you try to define two links with the same friendly text like this:

``` rst
See `here <http://microsoft.com>`_.
Now see `here <http://google.com>`_.
```

Built that and you get the warning: `Duplicate explicit target name: "here".` 

If you instead use a double-underscore then no reusable target is defined and you get a "one-off" hyperlink, which is typically what most people intended. See http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#embedded-uris-and-aliases for more details.

You are right though that we should get this clarified in our style guide. Really we should be using explicit link targets instead of in-lining URIs everywhere so that the raw text is easier to read, but for now I just want to avoid noisy build warnings.